### PR TITLE
Add icon for leisure=outdoor_seating

### LINKED ISSUE #3241: Add icon for leisure=outdoor_seating
https://wiki.openstreetmap.org/wiki/Tag:leisure=outdoor_seating
2k uses.

Icon proposal:

leisure outdoor_seating

### PR DESCRIPTION
**Pull request by Adamant36**
Adds icon for leisure=outdoor_seating closes #3241
https://www.openstreetmap.org/#map=19/43.81804/-111.78297 (node)
outdoor seating node
https://www.openstreetmap.org/#map=19/42.86389/-112.45208 (way)
outdoor seating way

### CODE REVIEW

**Comment by Adamant36** [comment]
@kocio-pl Any chance you could test and merge this? thanks.

**Comment by kocio-pl** [comment]
I think that z18+ might be better for such places, since this is usually a part of some other object, not a standalone object, so should not be shown too early.

**Comment by Adamant36** [comment]
Alright, that makes sense. I updated it.

File: project.mml [hunk 1]
```diff
@@ -1486,7 +1486,7 @@ Layer:
               'shop' || CASE WHEN shop IN ('no', 'vacant', 'closed', 'disused', 'empty') OR shop IS NULL THEN NULL ELSE '' END,
               'office' || CASE WHEN tags->'office' IN ('no', 'vacant', 'closed', 'disused', 'empty') OR (tags->'office') IS NULL THEN NULL ELSE '' END,
               'leisure_' || CASE WHEN leisure IN ('water_park', 'playground', 'miniature_golf', 'golf_course', 'picnic_table',
-                                                  'fitness_centre', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley') THEN leisure ELSE NULL END,
+                                                  'fitness_centre', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley', 'outdoor_seating') THEN leisure ELSE NULL END,
```
Comment (kocio-pl) [inline project.mml]: Please break this line, so it won't be the longest in this section.

File: project.mml [hunk 2]
```diff
@@ -1600,7 +1600,7 @@ Layer:
               'shop' || CASE WHEN shop IN ('no', 'vacant', 'closed', 'disused', 'empty') OR shop IS NULL THEN NULL ELSE '' END,
               'office' || CASE WHEN tags->'office' IN ('no', 'vacant', 'closed', 'disused', 'empty') OR (tags->'office') IS NULL THEN NULL ELSE '' END,
               'leisure_' || CASE WHEN leisure IN ('water_park', 'playground', 'miniature_golf', 'golf_course', 'picnic_table', 'slipway',
-                                                  'dog_park', 'fitness_centre', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley') THEN leisure ELSE NULL END,
+                                                  'dog_park', 'fitness_centre', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley', 'outdoor_seating') THEN leisure ELSE NULL END,
```
Comment (kocio-pl) [inline project.mml]: Please break this line, so it won't be the longest in this section.

File: project.mml [hunk 3]
```diff
@@ -2068,7 +2068,7 @@ Layer:
               'office' || CASE WHEN tags->'office' IN ('no', 'vacant', 'closed', 'disused', 'empty') OR (tags->'office') IS NULL THEN NULL ELSE '' END,
               'leisure_' || CASE WHEN leisure IN ('swimming_pool', 'water_park', 'miniature_golf', 'golf_course', 'fitness_centre', 'sports_centre', 'stadium', 'track',
                                                   'pitch', 'playground', 'park', 'recreation_ground', 'common', 'garden', 'nature_reserve', 'marina',
-                                                  'picnic_table', 'dog_park', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley') THEN leisure ELSE NULL END,
+                                                  'picnic_table', 'dog_park', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley', 'outdoor_seating') THEN leisure ELSE NULL END,
```
Comment (kocio-pl) [inline project.mml]: Please break this line, so it won't be the longest in this section.

File: project.mml [hunk 4]
```diff
@@ -2235,7 +2235,7 @@ Layer:
                   'office' || CASE WHEN tags->'office' IN ('no', 'vacant', 'closed', 'disused', 'empty') OR (tags->'office') IS NULL THEN NULL ELSE '' END,
                   'leisure_' || CASE WHEN leisure IN ('swimming_pool', 'water_park', 'miniature_golf', 'golf_course', 'fitness_centre', 'sports_centre', 'stadium', 'track',
                                                       'pitch','playground', 'park', 'recreation_ground', 'common', 'garden', 'nature_reserve', 'marina',
-                                                      'slipway', 'picnic_table', 'dog_park', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley') THEN leisure ELSE NULL END,
+                                                      'slipway', 'picnic_table', 'dog_park', 'fitness_station', 'firepit', 'sauna', 'beach_resort', 'bowling_alley', 'outdoor_seating') THEN leisure ELSE NULL END,
```
Comment (kocio-pl) [inline project.mml]: Please break this line, so it won't be the longest in this section.

Comment (Adamant36) [reply project.mml]: Done. I was thinking it was getting a little long.

**Comment by Adamant36** [comment]
@kocio-pl OK. Its done. I was thinking they were getting a little long.

**Comment by kocio-pl** [comment]
Did you push this new commit? I don't see any change.

**Comment by Adamant36** [comment]
I thought I had. It went through the Travis check and everything. So I don't know. It seems to be showing the changes now though.

**Comment by kocio-pl** [comment]
OK, now it's visible.

I have found [a place in the city](https://www.openstreetmap.org/node/4352954886#map=19/52.40549/16.92274), which tells me it should be further demoted to z19+ to not make a clutter in similar places - outdoor sitting might be small scale object:

z18
2jt3 dct

z19
cft3bwmh

**Comment by Adamant36** [comment]
I updated it to z19. Hopefully everything is good to go now.

**Comment by kocio-pl** [comment]
Thanks, now I think it's working good for me.

**Pull request merged**